docs/node/web3.mdx

```diff
@@ -209,6 +209,68 @@ service] and register it in the service manager for your platform.
 
 :::
 
+### Metrics
+
+Web3 Gateway can report a number of metrics to Prometheus server. By default,
+no metrics are collected and reported. There is one way to enable metrics
+reporting by configuring the `monitoring` section in the config file of your
+Web3 gateway.
+
+```yaml title='gateway.yml'
+...
+# Existing fields omitted ...
+gateway:
+# Existing fields omitted ...
+  monitoring:
+    # Change host to your external IP address if you have users accessing Web3 from the outside.
```

Comment:
Let's skip this comment. Since the monitoring server is not intended to be publicly accessible by default (e.g., without firewalls or an authentication layer), we shouldn't encourage it.

Anyone experienced enough will know how to configure it as needed.